#!/usr/bin/python3
def raise_exception():
    print(5 / "H")
